doc/api/stream.md

```diff
@@ -2807,7 +2807,7 @@ added:
 
 Returns whether the stream is readable.
 
-### `stream.Readable.toWeb(streamReadable)`
+### `stream.Readable.toWeb(streamReadable[, strategy])`
```

Comment:
I think we should use an options object here for future extensibility reason, i.e `{ strategy }`.